core/matrix/fbcsr.cpp

```diff
@@ -339,10 +341,9 @@ void Fbcsr<ValueType, IndexType>::read(const mat_data &data)
     index_type cur_bcol = blocks.begin()->first.block_column;
     const index_type num_brows = detail::get_num_blocks(bs, data.size[0]);
 
-    // blockutils::DenseBlocksView<value_type, index_type> values(
-    //     tmp->values_.get_data(), bs, bs);
-    range<accessor::col_major<value_type, 3>> values(
-        tmp->values_.get_data(), dim<3>(blocks.size(), bs, bs));
+    acc::range<acc::block_col_major<value_type, 3>> values(
+        std::array<size_type, 3>{blocks.size(), (size_type)bs, (size_type)bs},
+        tmp->values_.get_data());
```

Comment:
Please don't use c-style casts:
```suggestion
    acc::range<acc::block_col_major<value_type, 3>> values(
        std::array<size_type, 3>{blocks.size(), static_cast<size_type>(bs), static_cast<size_type>(bs)),
        tmp->values_.get_data());
```